projects/observability/src/shared/dashboard/widgets/top-n/top-n-widget-renderer.component.ts

```diff
@@ -74,18 +74,19 @@ export class TopNWidgetRendererComponent extends InteractiveDataWidgetRenderer<T
     this.entityNavService.navigateToEntity(item.entity);
   }
 
-  protected fetchData(): Observable<TopNData[]> {
+  protected fetchData(): Observable<SortNData[]> {
     return this.model.getData().pipe(
       tap(fetcher => {
         this.fetcher = fetcher;
         this.setInitialMetricSpecification(fetcher);
         this.setOptionsObservables(fetcher);
       }),
-      switchMap(() => this.buildDataObservable())
+      switchMap(() => this.buildDataObservable()),
+      map(data => data.sort((first, second) => second.value - first.value))
```

Comment:
So sort n doesn't sort? I think this pattern is right, it's just a naming thing. What do we think of `gauge-list`?

Reply:
renamed it to gauge-list